parsers/shared/src/main/scala/sigmastate/lang/ContractSignatureParser.scala

```diff
@@ -0,0 +1,27 @@
+package sigmastate.lang
+
+import fastparse._
+import fastparse.NoWhitespace._
+import sigmastate.SType
+
+case class ContractParam(name: String, tpe: SType, defaultValue: Option[SType#WrappedType])
+
+case class ContractSignature(name: String, params: Seq[ContractParam])
+
+object ContractSignatureParser {
+  import SigmaParser._
+
+  def parse(source: String): Parsed[ContractSignature] = {
+    fastparse.parse(source, parse(_))
+  }
```

Comment:
Same here, all the parsers can move to ContractParser and no need to ContractSignatureParser as independent facade. Or am I missing something?